numpy/__init__.pyi

```diff
@@ -1527,6 +1526,8 @@ class str_(character, str):
         self, __value: bytes, encoding: str = ..., errors: str = ...
     ) -> None: ...
 
+unicode_ = str_
```

Comment:
```suggestion
unicode_ = str0 = Str0 = str_
```
`str0` and `Str0` are two other aliases:
``` python
In [1]: import numpy as np

In [2]: {k: getattr(np, k) for k in np.__all__ if getattr(np, k) is np.str_}
Out[2]: 
{'str0': numpy.str_,
 'Str0': numpy.str_,
 'str_': numpy.str_,
 'unicode_': numpy.str_}
```

Reply:
Added `str0`